src/components/terms-modal.tsx

```diff
@@ -113,16 +113,13 @@ export default function TermsModal() {
                     ACCESS, USE, PARTICIPATE in the Airdrop. If you DO NOT AGREE
                     to these TnCs YOU SHALL BE PROHIBITED FROM CLAIMING THE
                     MENTO GOVERNANCE TOKEN(S). <br /> <br /> These Airdrop TnCs
-                    constitute the entire agreement between you and [issuing
```

Comment:
There's one more "by the issuing company" on line 108 that also needs to be removed

<img width="635" alt="image" src="https://github.com/mento-protocol/airgrab-interface/assets/117495/24728c82-a860-471d-a273-8d2f141d807d">